code/modules/mob/living/carbon/xenomorph/strains/castes/praetorian/valkyrie.dm

```diff
@@ -0,0 +1,101 @@
+/datum/xeno_strain/valkyrie
+	name = PRAETORIAN_VALKYRIE
+	description = "You trade your ranged abilities and acid to gain the ability to buff other Xenomorphs, giving them extra armor, as well as a dash that lets you dash to enemies to slice and root them. Targeting an ally causes you to dash towards them and knockdown the enemies around them. Lastly you get an ability that rejuvenates everyone in a certain rage depending on your rage."
```

Comment:
```suggestion
	description = "You trade your ranged abilities and acid to gain the ability to emit strong pheromones and buff other Xenomorphs, giving them extra armor, as well as a dash that lets you dash to enemies to slice and root them. Targeting an ally causes you to dash towards them and knockdown the enemies around them. Lastly you get an ability that rejuvenates everyone in a certain rage depending on your rage."
```
Completely forgot that base prae has no pheros when adding to this... base prae is so useless...